packages/editor-ui/src/App.vue

```diff
@@ -181,6 +181,21 @@ export default mixins(newVersions, showMessage, userHelpers).extend({
 				window.document.body.classList.add(`theme-${theme}`);
 			}
 		},
+		checkForExecutionsCount() {
+			let acc = 0;
+			const interval = setInterval(async () => {
+				try {
+					const plan = await this.usersStore.getOwnerCurrentPLan();
+					if (plan.metadata.slug !== 'trial-1') {
+						clearTimeout(interval);
+					}
+					// TODO: remove before releasing
+					plan.usage.executions += acc;
+					acc += 20;
+					this.usersStore.setCloudPLan(plan);
+				} catch {}
```

Comment:
we should clearTimeout here as well.. if userId does not exist, then we are not on a cloud instance..